qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst

```diff
@@ -1,32 +1,25 @@
 Processing recommendations
 ==========================
 
-Currently, Qiita supports the processing :sup:`(*)` of raw data from:
+Currently, Qiita supports the processing of raw data from:
 
 #. Target gene barcoded sequencing
 #. Shotgun sequencing
 
-Note that the selected processing are mainly guided so we can perform meta-analyses, this is combine different studies, even from different wet lab techniques or
-sequencing technologies. Remember to check the :ref:`example_study_processing_workflow` before continuing.
+Note that the selected processing are mainly guided so we can perform meta-analyses, this is combine different studies,
+even from different wet lab techniques or sequencing technologies.
 
-For more information about meta-analysis, examples and things to consider:
-
-- `"Tiny microbes, enormous impacts: what matters in gut microbiome studies?" <https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1086-x>`_
-- `"Meta-analyses of studies of the human microbiota" <http://genome.cshlp.org/content/23/10/1704.short>`_.
-- `"A Guide to Enterotypes across the Human Body: Meta-Analysis of Microbial Community Structures in Human Microbiome Datasets" <http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1002863>`_.
-- `"Dynamic changes in short- and long-term bacterial composition following fecal microbiota transplantation for recurrent Clostridium difficile infection" <http://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-015-0070-0>`_.
-
-:sup:`(*)` Remember that you can also upload BIOM tables for plotting but not covered here because this is only for raw data.
 
 Target gene barcoded sequencing
 -------------------------------
 
-For this you can start with raw, not demultiplexed data or per_sample_FASTQ, see :ref:`example_study_processing_workflow`. Either way, you will need to
-"Split libraries and QC", which uses the default in QIIME 1.9.1. Once your demultiplexed and QCed artifact is created you need to select which processing to perform.
-There are two main ideologies/methodologies to process target gene data: sequence clustering and sequence cleanup.
+For this you can start with raw, not demultiplexed data or per_sample_FASTQ. Either way, you will need to
+"Split libraries and QC", which uses the default in QIIME 1.9.1. Once your demultiplexed and QCed artifact is created
+you need to select which processing to perform. There are two main ideologies/methodologies to process target
+gene data: sequence clustering and sequence cleanup.
```

Comment:
I noticed that you replaced "cleanup" with "deblur" below. Should this place be also edited for consistency?

Reply:
thanks